### ISSUE TITLE
Quest still appears in party despite being complete

### ISSUE THREAD

**Issue by tildelowengrimm** [HabitRPG/habitica#2507]
Yesterday, on cron, I got a pop-up notification that I had completed a quest. I was able to find the quest mount and ride it. However, on the social page, the quest is still listed, and the boss still seems to have HP.

My user id is: `d704a894-509b-4bf6-a2a1-b6dd9e17fa36`
My group id is: `4b2733b2-4cac-4b2b-8a8e-8a0fa951ffec`
##

**Comment by deilann** [comment]
Is this still happening?

**Comment by deilann** [comment]
@flamsmark Please reopen if this is still an issue.

**Comment by ghost** [comment]
I am having a similar problem.  My party is doing the Egg Hunt quest, and today I saw the reward of 10 eggs in my inventory, but the Quest window still shows up in the Party screen with a progress of 98/100 eggs.  The eggs that I found yesterday don't show up in the Party Chat window (I'm assuming that this is related).  Also, when I try to use a Quest Scroll to start a new quest, I get an error message that my Party is already engaged in a quest.  Please help!

**Comment by jitspoe** [comment]
I'm having the exact same issue as NadsoBadso.  I received a notification that the quest is complete, and I have the eggs in my inventory, but under the party tab, it shows the quest is still incomplete with 97/100 eggs.  I was the only person in the group.

User ID: 32b750db-20f9-425e-93ea-7c47da945e34
Group ID: db73abcd-3bbf-4427-b20d-1b0a9d39bb12

...

Ok, scratch that.  After logging into github to make this post, I reloaded the HabitRPG page one more time, just to be sure, and it said it was completed.  I waited like a day before I decided to report this, since there seems to be some kind of delay.  When I first started the quest, I wasn't finding any eggs, then it started showing eggs found from like 5+ days ago.  The completion notification and inventory addition happened instantly when I marked a task as completed, but I guess the data in the party tab takes a few days to update.
